pom.xml

```diff
@@ -16,6 +16,19 @@
         <maven.compiler.target>9</maven.compiler.target>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.3.1</version>
+                <configuration>
+                    <outputDirectory>${project.basedir}/../GeneralService</outputDirectory>
```

Comment:
change this: ${project.basedir}/../GeneralService
to this: ${project.basedir}/../GeneralService/libs